Symfony/CS/Fixer/Symfony/PhpdocSeparationFixer.php

```diff
@@ -110,7 +110,7 @@ private function ensureAreTogether(DocBlock $doc, Annotation $first, Annotation
         $pos = $first->getEnd();
         $final = $second->getStart();
 
-        for (++$pos; $pos < $final; ++$pos) {
+        for ($pos = $pos + 1; $pos < $final; ++$pos) {
```

Comment:
Just to keep consistency with other places, where first `for` statement is assignment of iterator, and breaking that fools some IDEs and Scrutinizer.